### ISSUE TITLE
Convert paper numbers to use Animation

### ISSUE THREAD

**Issue by chrisklus** [phetsims/number-play#8]
Paper number animations are currently managed with custom code via `step`. If converted to use Animations, it would be easier to implement easing (so that they match the play objects animation style) and eliminate the need for calling `step` at all.

The best time to do this will likely be during the big refactor for #7.

**Comment by chrisklus** [comment]
All of the counting objects/paper numbers are animating really nicely now - I added quadratic easing for their positionProperty, as well as a scale property that animates between their play area size and stored size.

All that should be left now is cleanup of step functions since the counting objects no longer need to be stepped. This will get rid of a lot of code.

**Comment by chrisklus** [comment]
I cleaned up the step functions in the commits above. I also remembered I want to fix the animation target when the returning object has a handle on it - at the moment, the center position includes the bounds of the handle and it shouldn't

**Comment by chrisklus** [comment]
In the above commits I corrected the offset so a dropped paper number stack with a handle will animate to the correct place. I also animated the opacity of the handle down to zero on the way back so it doesn't jerkily disappear on top of the creator node.

I think that is enough work for this issues, at least for the prototype. The last two bits of "jerkiness" that may be nice to alleviate at some point in the future are:

1. When a card with, say 2 dogs on it, is returned to the creator node, it's not smooth that the full dog face suddenly appears when the 2 card is removed. (compare to the smoothness of dropping a single dog card on the creator node).
2. On the twenty screen, when a double digit card is returned to the creator node, the final dimensions of the animating card are still much wider than the creator node, so it's not smooth when that cards disappears (there is no double card target like in MAT).

Tagging @amanda-phet so your in the loop for the work done and my thoughts for problems still to solve (or not solve). I'm going to close this issue and open another one when it becomes relevant.
